src/main/java/org/jabref/gui/mergeentries/FetchAndMergeEntry.java

```diff
@@ -76,7 +78,16 @@ public void fetchAndMerge(BibEntry entry, List<Field> fields) {
                                   })
                                   .onFailure(exception -> {
                                       LOGGER.error("Error while fetching bibliographic information", exception);
-                                      dialogService.showErrorDialogAndWait(exception);
+                                      // client error
+                                      if (exception instanceof DOIDataNotFoundException) {
```

Comment:
Other JabRef code has the comments in the if:

```suggestion
                                      if (exception instanceof DOIDataNotFoundException) {
                                        // client error
```

Please also change at the other places

Reply:
OK, I'll change it later